### ISSUE TITLE
Duct leakage for single-zone CAV systems using zoneHVAC:AirDistributionUnit

### ISSUE THREAD

**Issue by rongpengzhang** [NREL/EnergyPlus#4988]
Duct leakage is not allowed for single-zone CAV systems using zoneHVAC:AirDistributionUnit

**Comment by mjwitte** [comment]
@rongpengzhang @hongtz68 @Myoldmopar @kbenne AirTerminal:SingleDuct:Uncontrolled (formerly known as "Direct Air") has always been inconsistent with all of the other terminal units - sits directly on the zone equipment list instead of being nested under an air distribution unit, has only one node, and the name (wonder who came up with that one . . . ) is confusing.  Given that the transition rule would be significant, I would suggest that we add a new terminal unit named AirTerminal:SingleDuct:ConstantVolume:NoReheat that has an inlet and outlet node and sits under an air distribution unit.  This would allow interfaces to implement the new object in a fashion consistent with the other terminal units, and then AirTerminal:SingleDuct:Uncontrolled could fade into disuse over time.

**Comment by kbenne** [comment]
Yes please! I have a special hate reserved for AirTerminalSingleDuctUncontrolled. OpenStudio does some shenanigans to make this look like it is part of the flow path, but because OS wants to reflect the nodes accurately it means we have this terminal plugged directly into the zone splitter.  Ick!

**Comment by kbenne** [comment]
And then we have to have special exceptions in translation to keep it off the air distribution unit.  I suspect every interface developer would or has had to deal with this.

**Comment by aaron-boranian** [comment]
**New single-duct constant volume air terminal without reheat**

The existing AirTerminal:SingleDuct:Uncontrolled is different from all other terminal units. It only has a single node (not an inlet and outlet) and it sits directly on the zone equipment list instead or being referenced by a ZoneHVAC:AirDistributionUnit object. This causes fits for interfaces, and it prevents this terminal unit from using the simple duct-leakage features in ZoneHVAC:AirDistributionUnit.

Propose adding a new terminal unit type AirTerminal:SingleDuct:ConstantVolume:NoReheat that has an inlet and outlet node and sits under an air distribution unit. Then deprecate the old AirTerminal:SingleDuct:Uncontrolled object.

Migrated from [UserVoice](http://energyplus.uservoice.com/forums/258860/suggestions/20306446) feedback from @mjwitte
